README.md

```diff
@@ -67,6 +67,7 @@ A categorized community-driven collection of awesome Pharo libraries, tools, fra
 + [TinyLogger](https://github.com/jecisc/TinyLogger) - A really small logger for Pharo applications.
 
 ## Miscellaneous
++ [ISO3166](https://github.com/hernanmd/ISO3166).
```

Comment:
The small description sentence is missing (in Units as well but I will fix it later).

Reply:
A short description for the ISO 3166-1 package:

Codes for the names of countries, dependent territories, and special areas of geographical interest for Pharo applications.

Reply:
I could add it but later, otherwise feel free to paste the above description